lua/wire/server/wirelib.lua

```diff
@@ -1219,6 +1211,14 @@ function WireLib.GetVersion()
 		end
 	end
 
+	-- Check if we're Workshop version first
```

Comment:
whoops, I missed to also change the comment to 'last' when I did copy&paste.
Should I make a new commit just to fix this wrong word?